JS/script.js

```diff
@@ -16,8 +16,30 @@ async function getWeather(city){
     document.querySelector(".humidity").innerHTML = Math.round(data.main.humidity) + "%"
     document.querySelector(".wind").innerHTML = Math.round(data.wind.speed)+"km/h"
     document.querySelector(".description").innerHTML = data.weather[0].description
-
- if(data.weather[0].main == "Clear"){
+    document.querySelector(".feels-like").innerHTML = "RealFeel     "+Math.round(data.main.feels_like) + "°"
+    document.querySelector(".pressure").innerHTML = "Pressure:  "+Math.round(data.main.pressure) + "Br"
+    document.querySelector(".visibilty").innerHTML = "Visibilty:  "+data.visibility/1000 + "Km"
+    console.log(data.sys.sunrise)
+// convert unix time stamp to hours and minutes for sunrise
+   const unixTimestamp = data.sys.sunrise;
+   const sunrise = unixTimestamp *1000;
+   const date = new Date(sunrise)
+   const amOrPm = date.getHours() >= 12 ? 'PM' : 'AM';
+    document.querySelector("#sunrise").innerHTML = "Sunrise:  "+ date.getHours() +":"+date.getMinutes() + "  "+amOrPm
+    document.querySelector(".cloud-cover").innerHTML = "Cloud Cover:  "+data.clouds.all + "%"
+   
+ 
+ // convert unix time stamp to hours and minutes for sunset
+ const unixTimestamp1 = data.sys.sunset;
+ const sunset = unixTimestamp1 *1000;
+ const dates = new Date(sunset)
+ const amOrPms = date.getHours() >= 12 ? 'PM' : 'AM';
+ 
+ document.querySelector(".sunset").innerHTML = "Sunset:  "+ dates.getHours() +":"+dates.getMinutes() + "  "+amOrPm
+ 
+ 
+ 
+    if(data.weather[0].main == "Clear"){
```

Comment:
Please apply strict comparison by converting both values to lowercase and trimming them prior to the comparison. This practice enhances the accuracy of the evaluation.